client/state/ui/guided-tours/actions.js

```diff
@@ -21,14 +21,16 @@ import { getPreference } from 'state/preferences/selectors';
  * @param {Object} options Options object, see fn signature.
  * @return {Object} Action object
  */
-export function showGuidedTour( { shouldShow, shouldDelay = false, tour = 'main' } ) {
+export function showGuidedTour( { shouldShow, shouldDelay = false, tour } ) {
 	const showAction = {
 		type: GUIDED_TOUR_SHOW,
 		shouldShow,
 		shouldDelay,
 		tour,
 	};
 
+	// TODO(mcsf): track this *somewhere*, now that we'll stop relying on this
+	// action for launching tours
```

Comment:
Discussing this with @lsinger — there would be more than one way. The bare minimum would be to _cheat_ and still manually dispatch something in `boot` for the purpose of analytics, which something more sophisticated is conceived.

For instance, if we were to follow the route of implementing some assistant GT middleware, that could be a piece responsible for analytics. A less sophisticated alternative would be to let the view layer (perhaps in `componentDidUpdate`) look for cases where `stepName === 'init'` and decide to dispatch a `calypso_guided_tours_show` event — caveat would be making sure we don't dispatch duplicate events if something unexpected happens with the props.